tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderEntity.cs

```diff
@@ -1,99 +1,99 @@
-                @OrganisationContactId,
-                @OrderStatusId,
-                @Created,
-                @LastUpdated,
-                @LastUpdatedBy,
-                @LastUpdatedByName
-            );";
-
+﻿using System;
+using System.Threading.Tasks;
+
+namespace NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Entities
+{
+    public sealed class OrderEntity : EntityBase
+    {
+        public string OrderId { get; set; }
+
+        public string Description { get; set; }
+
+        public Guid OrganisationId { get; set; }
+
+        public string OrganisationName { get; set; }
+
+        public string OrganisationOdsCode { get; set; }
+
+        public int? OrganisationAddressId { get; set; }
+
+        public int? OrganisationBillingAddressId { get; set; }
+
+        public int? OrganisationContactId { get; set; }
+
+        public int OrderStatusId { get; set; }
+
+        public DateTime Created { get; set; }
+
+        public DateTime LastUpdated { get; set; }
+
+        public Guid LastUpdatedBy { get; set; }
+
+        public string LastUpdatedByName { get; set; }
+
+        protected override string InsertSql => @"
+            INSERT INTO dbo.[Order]
+            (
+                OrderId,
+                Description,
+                OrganisationId,
+                OrganisationName,
+                OrganisationOdsCode,
+                OrganisationAddressId,
+                OrganisationBillingAddressId,
+                OrganisationContactId,
+                OrderStatusId,
+                Created,
+                LastUpdated,
+                LastUpdatedBy,
+                LastUpdatedByName
+            )
+            VALUES
+            (
+                @OrderId,
+                @Description,
+                @OrganisationId,
+                @OrganisationName,
+                @OrganisationOdsCode,
+                @OrganisationAddressId,
+                @OrganisationBillingAddressId,
+                @OrganisationContactId,
+                @OrderStatusId,
+                @Created,
+                @LastUpdated,
+                @LastUpdatedBy,
+                @LastUpdatedByName
+            );";
+
         public static async Task<string> FetchOrderDescriptionFromOrderId(string connectionString, string orderId)
-        {
-            return (await SqlRunner.QueryFirstAsync<string>(connectionString, $@"SELECT
-                         [Description]
-                         FROM [Order]
+        {
+            return (await SqlRunner.QueryFirstAsync<string>(connectionString, $@"SELECT
+                         [Description]
+                         FROM [Order]
+                         WHERE [OrderId] = @orderId", new { orderId }));
+        }
+
+        public static async Task<string> FetchLastUpdatedByNameFromOrderId(string connectionString, string orderId)
+        {
+            return (await SqlRunner.QueryFirstAsync<string>(connectionString, $@"SELECT

```

Comment:
As we're not actually doing any string interpolation here we can omit the `$`.

Reply:
👍